libs/platform/core/src/services/hydration/default-hydration.service.ts

```diff
@@ -13,6 +13,8 @@ import { ComponentsPlugin } from '../../orchestration/components';
 import { HydrationService, HydrationTrigger } from './hydration.service';
 import { ContextService } from '../context';
 
+export const visibilityAttribute = 'dynamic-visibility';
```

Comment:
Wondering, if we could reuse existing hydratable directive, with some additional new value. Take a look also at changes here: https://github.com/spryker/oryx/pull/602